src/cls/rgw/cls_rgw_ops.h

```diff
@@ -234,6 +234,8 @@ WRITE_CLASS_ENCODER(rgw_cls_link_olh_op)
 struct rgw_cls_unlink_instance_op {
   cls_rgw_obj_key key;
   std::string op_tag;
+  // Q: does this represent a remote epoch during multisite sync?
+  // so for an unlink op coming from the same zone this will always be 0?
   uint64_t olh_epoch;
```

Comment:
`olh_epoch = 0` does seem to be the case for `RGWDeleteObj` in general. there seems to be a [special case for system requests](https://github.com/ceph/ceph/blob/86aba97a/src/rgw/rgw_op.cc#L5537-L5548) that provide a `rgwx-versioned-epoch` header, but i don't think that's used anywhere?

Reply:
can these questions be removed or turned into statements?